### PR TITLE
chore(components): use svg circle for the app loader

### PR DESCRIPTION
**Pull request by ybaskaran**
**What is the problem this PR is trying to solve?**
AppLoader spinner animation should be improved

**What is the chosen solution to this problem?**
Use svg circle for a better spinning animation instead of plain div loader
References: http://guidelines.talend.com/document/92132#/messaging-communication/loading-feedback
https://jira.talendforge.org/browse/TUI-49

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/AppLoader/constant.js [hunk 1]
```diff
@@ -44,42 +26,73 @@ body {
...
+	margin: -27px 0 0 -27px;
+	animation: rotate 2s linear infinite;
 }
 
-.tc-app-loader:before {
-	animation-delay: -0.3s;
+.tc-app-loader-path {
+	stroke-dasharray: 12.5664; 
+	stroke-dashoffset: 0;
+	stroke: #ababab;
```
Comment (jsomsanith) [inline packages/components/src/AppLoader/constant.js]: This is `$silver-chalice`

File: packages/components/src/AppLoader/constant.js [hunk 2]
```diff
@@ -44,42 +26,73 @@ body {
...
-	animation-delay: -0.15s;
+@keyframes rotate {
+	100% {
+		transform: rotate(360deg);
+	}
+}
+
+@keyframes dash {
+	0% {
+	margin: -7px 0 0 -7px;
```
Comment (jsomsanith) [inline packages/components/src/AppLoader/constant.js]: Indentation issue

File: packages/components/src/AppLoader/constant.js (hunk 1 above)
Comment (ybaskaran) [reply packages/components/src/AppLoader/constant.js]: we cannot use scss variables as this is a plain css

File: packages/components/src/AppLoader/constant.js (hunk 2 above)
Comment (ybaskaran) [reply packages/components/src/AppLoader/constant.js]: ah, its a typo, removed

**Pull request closed without merging**